src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/Contracts/IBrowserRefreshServerAccessor.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.HotReload;
+
+public interface IBrowserRefreshServerAccessor : IDisposable
+{
+    ValueTask StartServerAsync(CancellationToken cancellationToken);
```

Comment:
It can be removed, it was mean to provide an interface version of AbstractBrowserRefreshServerAccessor contract in the earlier attempt but no longer needed